doc/manual/tags.md

```diff
@@ -3,96 +3,296 @@
 ## Base package tags
 
 * Name (1000) string
+
+  Package name.
+
 * Version (1001) string
+
+  Package version.
```

Comment:
Shouldn't it provide even more information or example? E.g. "This typically corresponds to upstream package version."

Reply:
Actually, there is some explanation in https://github.com/rpm-software-management/rpm/blob/master/doc/manual/queryformat.md#tags if I understand this, so this could be interlinked.

Reply:
Look, this is not a finished book which will go to press and never revisited again. It's about gradually doing *at least something*, and if I start worrying about making it perfect, it's simply going to stay in the entirely undocumented state forever.